actors/miner/tests/util.rs

```diff
@@ -2549,15 +2549,16 @@ fn fixed_hasher(offset: ChainEpoch) -> Box<dyn Fn(&[u8]) -> [u8; 32]> {
 }
 
 #[allow(dead_code)]
-pub fn test_sector(
-    expiration: ChainEpoch,
+pub fn test_sector_no_proof_exp(
```

Comment:
Unnecessary to rename this, tho you might need another version with a proof expiration argument. And maybe add a year rather than 1000 epochs.

Reply:
yeah, that was the point, to allow for graceful addition of it